src/components/AppLayout/index.tsx

```diff
@@ -42,7 +42,8 @@ import {
 	ApplayoutIdentityIcon,
 	ArchivedIcon,
 	ClearIdentityOutlinedIcon,
-	RoundedDollarIcon
+	RoundedDollarIcon,
```

Comment:
don't import icon from custom icon and don't initialize there 
use <Image and dark-icons class name for dark mode

Reply:
tried but it was breaking in dark mode